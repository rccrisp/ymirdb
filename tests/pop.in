set a 1 2 3
pop a
pop a
pop a
pop a
set b 4 5 6
append a b
pop a
forward a
backward b
push a b
pop a
forward a
backward a
bye