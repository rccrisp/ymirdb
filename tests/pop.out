> ok

> 1

> 2

> 3

> nil

> ok

> ok

> b

> nil

> nil

> ok

> b

> nil

> nil

> bye
